datafusion/physical-plan/src/aggregates/topk/hash_table.rs

```diff
@@ -433,8 +440,27 @@ pub fn new_hash_table(
 #[cfg(test)]
 mod tests {
     use super::*;
+    use arrow::array::TimestampMillisecondArray;
+    use arrow_schema::TimeUnit;
     use std::collections::BTreeMap;
 
+    #[test]
+    fn should_emit_correct_type() -> Result<()> {
```

Comment:
I verified this test fails without the code change in this PR and thus conclude the test covers the change

```

assertion `left == right` failed
  left: Timestamp(Millisecond, None)
 right: Timestamp(Millisecond, Some("UTC"))

Left:  Timestamp(Millisecond, None)
Right: Timestamp(Millisecond, Some("UTC"))
<Click to see difference>
```

Reply:
Thanks. I started with @niebayes ' test and created this one instead because it covered the case and felt less like an e2e test.